bundles/net.tourbook.device.garmin.fit/src/net/tourbook/device/garmin/fit/IPreferences.java

```diff
@@ -29,11 +29,11 @@
 
    String FIT_PREFERRED_POWER_DATA_SOURCE    = "FIT_POWER_DATA_SOURCE";              //$NON-NLS-1$
 
-   String FIT_IMPORT_TOURTYPE                = "FIT_IMPORT_TOURTYPE";               //$NON-NLS-1$
-   String FIT_IMPORT_TOURTYPE_MODE           = "FIT_IMPORT_TOURTYPE_MODE";          //$NON-NLS-1$
+   String FIT_IS_IMPORT_TOURTYPE                = "FIT_IS_IMPORT_TOURTYPE";               //$NON-NLS-1$
+   String FIT_IS_IMPORT_TOURTYPE_MODE           = "FIT_IS_IMPORT_TOURTYPE_MODE";          //$NON-NLS-1$
 
-   String FIT_IMPORT_TOURTYPE_MODE_SPORT           = "FIT_IMPORT_TOURTYPE_MODE_SPORT";              //$NON-NLS-1$
-   String FIT_IMPORT_TOURTYPE_MODE_PROFILE         = "FIT_IMPORT_TOURTYPE_MODE_PROFILE";            //$NON-NLS-1$
-   String FIT_IMPORT_TOURTYPE_MODE_TRYPROFILE      = "FIT_IMPORT_TOURTYPE_MODE_TRYPROFILE";         //$NON-NLS-1$
-   String FIT_IMPORT_TOURTYPE_MODE_SPORTANDPROFILE = "FIT_IMPORT_TOURTYPE_MODE_SPORTANDPROFILE";    //$NON-NLS-1$
+   String FIT_IS_IMPORT_TOURTYPE_MODE_SPORT           = "FIT_IS_IMPORT_TOURTYPE_MODE_SPORT";              //$NON-NLS-1$
+   String FIT_IS_IMPORT_TOURTYPE_MODE_PROFILE         = "FIT_IS_IMPORT_TOURTYPE_MODE_PROFILE";            //$NON-NLS-1$
+   String FIT_IS_IMPORT_TOURTYPE_MODE_TRYPROFILE      = "FIT_IS_IMPORT_TOURTYPE_MODE_TRYPROFILE";         //$NON-NLS-1$
+   String FIT_IS_IMPORT_TOURTYPE_MODE_SPORTANDPROFILE = "FIT_IS_IMPORT_TOURTYPE_MODE_SPORTANDPROFILE";    //$NON-NLS-1$
```

Comment:
`FIT_IMPORT_TOURTYPE` --> `FIT_IS_IMPORT_TOURTYPE ` this is OK

`FIT_IMPORT_TOURTYPE_MODE_...`  --> `FIT_IS_IMPORT_TOURTYPE_MODE_...` this was not my suggestion as this is not a boolean value

Reply:
My misunderstanding, it's fixed